os/arch/Kconfig.chip

```diff
@@ -85,6 +86,10 @@ config ARCH_HAVE_RESET
 	bool
 	default n
 
+config ARCH_HAVE_SETJMP
```

Comment:
Will you give a choice to user to enable/disable setjmp?
I don't know why you added this.

Reply:
If you give option then you need to auto resolve all dependencies when the option is disabled and make sure build doesn't break.

Reply:
It is not for the user. It is a "hardware" feature that TizenRT may or may not provide, depending on the architecture. TizenRT maintainer (you) manage this flag for each architecture.
For armv-a, -r, -m we do provide t, for anything else we don't (yet).

That's why apps/examples/setjmp depends on ARCH_HAVE_SETJMP - if it is not provided, the example app would not be available.
If you do not have this flag (which is an option), the compile would crash with errors if an app requires setjmp/longjmp. If you have this flag, menuconfig would not allow to configure such application.